src/tests/functional/plugin/shared/src/single_layer_tests/op_impl_check/single_op_graph.cpp

```diff
@@ -36,21 +557,201 @@ std::shared_ptr<ov::Model> generateBinaryEltwise(const std::shared_ptr<ov::op::O
+    } else {
+        return nullptr;
+    }
+
+    ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwiseNode)};
+    return std::make_shared<ov::Model>(results, params, "BinaryEltwiseGraph");
+}
+
+std::shared_ptr<ov::Model> generateBinaryEltwiseComp(const std::shared_ptr<ov::op::Op>& node) {
+    const auto params = ngraph::builder::makeDynamicParams(ov::element::f32, {{2}, {2}});
+    std::shared_ptr<ov::Node> eltwise;
+    if (ov::is_type<ov::op::v1::Equal>(node)) {
+        eltwise = std::make_shared<ov::op::v1::Equal>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::Greater>(node)) {
+        eltwise = std::make_shared<ov::op::v1::Greater>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::GreaterEqual>(node)) {
+        eltwise = std::make_shared<ov::op::v1::GreaterEqual>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::Less>(node)) {
+        eltwise = std::make_shared<ov::op::v1::Less>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::LessEqual>(node)) {
+        eltwise = std::make_shared<ov::op::v1::LessEqual>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::NotEqual>(node)) {
+        eltwise = std::make_shared<ov::op::v1::NotEqual>(params[0], params[1]);
+    } else {
+        return nullptr;
+    }
+
+    ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwise)};
+    return std::make_shared<ov::Model>(results, params, "BinaryEltwiseComparisonGraph");
+}
+
+std::shared_ptr<ov::Model> generateBinaryEltwiseLogical(const std::shared_ptr<ov::op::Op>& node) {
+    const auto params = ngraph::builder::makeDynamicParams(ov::element::boolean, {{1}, {1}});
+    std::shared_ptr<ov::Node> eltwise;
+    if (ov::is_type<ov::op::v1::LogicalAnd>(node)) {
+        eltwise = std::make_shared<ov::op::v1::LogicalAnd>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::LogicalOr>(node)) {
+        eltwise = std::make_shared<ov::op::v1::LogicalOr>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v1::LogicalXor>(node)) {
+        eltwise = std::make_shared<ov::op::v1::LogicalXor>(params[0], params[1]);
+    } else if (ov::is_type<ov::op::v0::Xor>(node)) {
+        eltwise = std::make_shared<ov::op::v0::Xor>(params[0], params[1]);
+    } else {
+        return nullptr;
+    }
+
+    ov::ResultVector results{std::make_shared<ov::op::v0::Result>(eltwise)};
+    return std::make_shared<ov::Model>(results, ngraph::ParameterVector{params}, "BinaryEltwiseLogicalGraph");
+}
+
+std::shared_ptr<ov::Model> generateBroadcast(const std::shared_ptr<ov::op::Op>& node) {
+    const ov::Shape input_shape{};
+    const ov::Shape output_shape{5, 4, 3, 2};
+    const auto params = std::make_shared<ov::op::v0::Parameter>(ov::element::f32, input_shape);
+    const auto shape_const =
+        ov::op::v0::Constant::create(ov::element::u64, ov::Shape{output_shape.size()}, output_shape);
```

Comment:
Let's use single way to create constant node in the PR - builder::makeConstant().

Reply:
I'll update by applying `ngraph::builder::makeConstant()`